src/tensorflow.cc

```diff
@@ -1204,86 +1204,89 @@ ModelInstanceState::Create(
             std::string("CPU Execution Accelerator is not supported in "
                         "TensorFlow backend"));
 
-        RETURN_ERROR_IF_TRUE(
-            gpu_device == ModelInstanceState::NO_GPU_DEVICE,
-            TRITONSERVER_ERROR_INVALID_ARG,
-            std::string(
-                "GPU Execution Accelerator can only be set on non-CPU backend "
-                "context"));
-
-        triton::common::TritonJson::Value gpu_eas;
-        if (eas.Find("gpu_execution_accelerator", &gpu_eas)) {
-          for (size_t ea_idx = 0; ea_idx < gpu_eas.ArraySize(); ea_idx++) {
-            triton::common::TritonJson::Value ea;
-            RETURN_IF_ERROR(gpu_eas.IndexAsObject(ea_idx, &ea));
-            std::string name;
-            RETURN_IF_ERROR(ea.MemberAsString("name", &name));
-            if (name == kTensorRTExecutionAccelerator) {
-              // Validate and set parameters
-              triton::common::TritonJson::Value params;
-              if (ea.Find("parameters", &params)) {
-                std::vector<std::string> param_keys;
-                RETURN_IF_ERROR(params.Members(&param_keys));
-                for (const auto& param_key : param_keys) {
-                  std::string value_string;
-                  if (param_key == "precision_mode") {
-                    RETURN_IF_ERROR(params.MemberAsString(
-                        param_key.c_str(), &value_string));
-                    if (value_string == "FP32") {
-                      tftrt_config.precision_mode_ = TRITONTF_MODE_FP32;
-                    } else if (value_string == "FP16") {
-                      tftrt_config.precision_mode_ = TRITONTF_MODE_FP16;
+        // GPU Execution Accelerator is disabled on CPU devices.
+        if (gpu_device == ModelInstanceState::NO_GPU_DEVICE) {
+          LOG_MESSAGE(
+              TRITONSERVER_LOG_WARN,
+              "GPU Execution Accelerator will be ignored on CPU backend "
```

Comment:
"... will be ignored for model instance on CPU"

Reply:
Done